src/Microsoft.Azure.WebJobs.Shared/Sanitizer.cs

```diff
@@ -66,8 +66,10 @@ internal static string Sanitize(string input)
                         break;
                     }
 
+                    int startIndexAfterToken = startIndex + token.Length;
+
                     // Find the end of the secret. It most likely ends with either a double quota " or tag opening <
-                    int credentialEnd = t.IndexOfAny(ValueTerminators, startIndex);
+                    int credentialEnd = t.IndexOfAny(ValueTerminators, startIndexAfterToken);
```

Comment:
The current sanitizer code does not catch the scenarios such as `"key": "value"`. Since it looks for ValueTerminator `"` and ends up giving output `[Hidden Credential]: "value"`. Hence, looking for value with start index after token `key`.

Reply:
We have a followup item to improve this class overall - https://github.com/Azure/azure-functions-host/issues/10383

Reply:
I'm thinking that for dealing with customer thrown exceptions, it might be better for us to have a more comprehensive approach, since attempting to identify all the possible signatures customers may log is impossible. For example, you as a customer might log an exception with a message "Account Info {userName}, {pwd}" which token scanning won't identify.

Instead, I think we're going to need to identify all the places where we're invoking user code and handling exceptions, and wrap or mark those exceptions somehow such that later in the logging pipeline when they're logged we can completely avoid logging the exception details (perhaps we can log exception Type, etc.).